hugo.toml

```diff
@@ -180,35 +180,36 @@ js = [
 ]
 
 [[params.versions]]
-version = "v1.27"
-githubbranch = "v1.27.0"
+fullversion = "v1.28.0"
```

Comment:
I don't expect to see a `fullversion` param.

Reply:
Ah, sorry again for missing this 😓